core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/ArtifactClearingState.java

```diff
@@ -31,11 +31,21 @@ public String getFactContentName() {
     }
 
     public enum ClearingState {
-        INITIAL,
-        EXTERNAL_SOURCE,
-        AUTO_EXTRACT,
-        PROJECT_APPROVED,
-        OSM_APPROVED,
-        WORK_IN_PROGRESS;
+        INITIAL(1),
+        WORK_IN_PROGRESS(2),
+        EXTERNAL_SOURCE(3),
+        AUTO_EXTRACT(4),
+        PROJECT_APPROVED(5),
+        OSM_APPROVED(6);
```

Comment:
Is the order really the criterion to decide whether a release can be updated? I would rather think, there are clearing states in which updates are possible, and those in which this is not the case. So, should there just be a boolean field like _updatesAllowed_? Also, is the order meaningful for all states, e.g. does it make sense to compare _EXTERNAL_SOURCE_ with _AUTO_EXTRACT_?